File: Framework/CSharp/Framework/Framework/Web/Mvc/SmartTextResult.cs
/*
 * 项目：Smartkernel.Framework
 * 作者：曹艳白
 * 邮箱：caoyanbai@139.com
 */
using System.Web.Mvc;

namespace Smartkernel.Framework.Web.Mvc
{
    /// <summary>
    /// Text结果
    /// </summary>
    public class SmartTextResult : ContentResult
    {
        /// <summary>
        /// 构造函数
        /// </summary>
        public SmartTextResult()
            : base()
        {
            this.ContentType = SmartMimeType.Text;
        }
    }
}

File: Php/framework/framework/web/mvc/smarttextresult.php
<?php

namespace Smartkernel\Framework\Web\Mvc {

    use Smartkernel\Framework\Web\SmartMimeType;
    use Smartkernel\Framework\Web\Mvc\SmartContentResult;

/*
     * Text结果
     */

    class SmartTextResult extends SmartContentResult {
        /*
         * 构造函数
         */

        function __construct() {
            parent::__construct();
            $this->ContentType = SmartMimeType::Text;
        }

    }

}